src/Google.Pubsub.V1/project.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "version": "0.1.0-*",
+  "summary": "GAPIC library for Google.Pubsub.V1",
+  "description": "GAPIC library for Google.Pubsub.V1, making common operations simpler in client code.",
```

Comment:
There's nothing really for it to be simpler than - I'd just duplicate the summary.